governance/tag-governance.md

```diff
@@ -0,0 +1,541 @@
+standards bodies, such as the [Internet Engineering Task Force (IETF)] and the [Kubernetes project].
+
+- **MUST** - This word, or the terms "**REQUIRED**" or "**SHALL**," means that the item is required.  
+- **MUST NOT** - This phrase, or the phrase "**SHALL NOT**", means that the item is not permitted
+  or must be avoided.  
+- **SHOULD** -  This word, or the adjective "**RECOMMENDED**," means the item is suggested but may
+  be ignored for valid reasons.  
+- **SHOULD NOT** - This phrase, or the phrase "**NOT RECOMMENDED**," means the item is acceptable
+  under certain circumstances.  
+- **MAY** - This word, or the adjective "**OPTIONAL**," means the item is truly optional.
+
+
+[RFC2119 keywords]: https://www.ietf.org/rfc/rfc2119.txt
+[Internet Engineering Task Force (IETF)]: https://en.wikipedia.org/wiki/Internet_Engineering_Task_Force
+[Kubernetes project]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance-requirements.md
+
+## Technical Advisory Groups (TAGs)
+
+A CNCF TAG will oversee and coordinate the interests pertaining to a logical area of needs across
+projects, other CNCF groups, and users. They are long-lived groups that report directly to the TOC.
+Areas covered by TAGs may be vertically focused on industry problem domains that align with the
+projects within the CNCF and/or horizontal that support the other TAGs or the projects themselves.
+
+**Examples:**
+- **Vertical:** Infrastructure, Workloads  
+- **Horizontal:** Security & Compliance, Operational Resilience
+
+### Goals
+
+- Strengthen the project ecosystem to meet the needs of end users and project contributors.  
+- Serve as a “bridge” between projects to surface common problems and drive alignment.  
+- Identify gaps in the CNCF project portfolio. Find and attract projects to fill these gaps.  
+- Reduce some project workload on TOC while retaining executive control & tonal integrity with this
+  elected body.  
+- Provide technical expertise during the evaluation of projects to join the CNCF and as they move
+  levels.  
+- Educate and inform users with unbiased, effective, and practically useful information scoped to
+  cloud native.  
+- Focus attention & resources on helping foster project maturity, systematically across CNCF
+  projects.
+
+### Requirements
+
+- MUST have a charter that outlines scope, potential overlaps with other TAGs, and any operational
+  deviations from TAGs.  
+- MUST enumerate any roles within the TAG and the responsibilities of each along with any role
+  membership requirements and lifecycle.  
+- MUST maintain TAG “metadata” - a list of leads, subprojects, and initiatives.  
+- Where applicable, MUST participate in Project Reviews & Contributor Strategy TOC subprojects.  
+- If applicable, MUST document services provided to other TAGs or projects.  
+- MUST hold at least one regularly scheduled *public* monthly meeting, recorded and uploaded to
+  YouTube.  
```

Comment:
Will this be handled automatically by the LFX meetings that we're already using?

Reply:
they should be, yes.